### ISSUE TITLE
[Bug] Edge Deflector doesn't show up in settings

### ISSUE THREAD

**Issue by Andrei12333** [da2x/EdgeDeflector#156]
Please follow the troubleshooting steps first!
https://github.com/da2x/EdgeDeflector/wiki/Troubleshooting

**To Reproduce**
1. Go to Default apps for protocol
2. Scroll down to MICROSOFT-EDGE
3. Click default app

**Expected behavior**
For Edge deflector to show in the list

**Actual behavior/describe the bug:**
A clear and concise description of what happen instead.
Edge deflector didn't show up in the list

**Details:**
 - Windows version: Windows 10 Version 20H2 (OS Build 19042.1387)

**Screenshots:**
Bug:
![image](https://user-images.githubusercontent.com/72381315/144638586-0841f5c4-f796-4035-85c0-cc32932b44b8.png)
Winver:
![image](https://user-images.githubusercontent.com/72381315/144638935-b64b5875-06b1-4aef-818a-b941394d6720.png)

**Comment by Andrei12333** [comment]
An related issue: #147

**Comment by Zankoas** [comment]
Also getting this issue on windows build 19044.1387, happy to provide more information if need but is otherwise the same as the above

**Comment by stevenzhu25** [comment]
Is this possibly related to #155 ?

**Comment by 0x0lli** [comment]
Having that very same issue.
build 19043.1387
Windows Feature Experience Pack 120.2212.3920.0

**Comment by lucafossen** [comment]
Same issue here. I just got this issue after updating yesterday, now on 21H2 Windows Feature Experience Pack 120.2212.3920.0. Here is a screenshot of my update history if it helps:
![image](https://user-images.githubusercontent.com/62906154/145042511-3ceda760-c948-4b2a-b415-2b199cd738f6.png)

**Comment by lucafossen** [comment]
Same issue here. I just got this issue after updating yesterday, now on Windows 10 Pro 21H2 Windows Feature Experience Pack 120.2212.3920.0 build 19044.1387 Here is a screenshot of my update history if it helps:
![image](https://user-images.githubusercontent.com/62906154/145042511-3ceda760-c948-4b2a-b415-2b199cd738f6.png)

**Comment by MrSco** [comment]
same issue here. Windows 10 Pro 21H2 (OS Build 19044.1387)

**Comment by King-0f-Hero** [comment]
I'm on Windows 10 Home 21H2 Build 19044.1387 and having the same issue, which started only after I applied the update to this version. Before, everything was fine.

**Comment by griffi-gh** [comment]
I have the same issue after 21h2 update

**Comment by Shadow0013HUN** [comment]
I am facing the same issue since Windows 10 Pro 1703. I have reported it then, but my issue report was immediately discarded and the dev was extremely rude.

Guess he still did not fix this software :)

**Comment by lucafossen** [comment]
> I am facing this same exact issue described above since Windows 10 Pro 1703. I have reported it then, but my issue report was immediately discarded and the dev was extremely rude.
> 
> Guess he still did not fix this software :)

To be fair, it's not the dev's fault. Microsoft has blocked his software.

**Comment by da2x** [comment]
Duplicate of #141.
